wiki/Mascots/de.md

```diff
@@ -1,290 +1,101 @@
-
-#### Alias
-
-Don, Don-chan
-
-#### Geburtsdatum
-
-?
+> Don ist der Protagonist der Reihe [Taiko no Tatsujin](https://de.wikipedia.org/wiki/Taiko_no_Tatsujin). Er ist eine Taiko-Trommel mit weißem Rand und vier Beinen, einem roten Gesicht (das aussieht wie Moly Orange) und einem hellblauen Körper. Dons Traum ist es, die Schönheit von Taiko mit der Welt zu teilen. Drei Jahre sind vergangen, seit er im Wada-Haus eingezogen ist, und er ist in der Stadt ziemlich beliebt geworden. Er hat einen monströsen Appetit und macht manchmal teure Einkäufe im Wada-Haus, die manchmal das Schlimmste befürchten lassen. Er beendet seine Sätze gewöhnlich mit "Ta-don", was auf Japanisch "Ba-dum" bedeutet.
 
-#### Größe
+和田どん (Wada Don), auch bekannt als Don oder Don-chan, war eines der Maskottchen von osu! für osu!taiko, das seit Mai 2008 dabei ist. Er ist 48 Zentimeter groß und wiegt mehr als 100 Kilogramm. Er erscheint im Skin von osu!taiko. Sein Design stammt von Yukiko Yokoo (横尾有希子) und seine Stimme wurde von Narahashi Miki (楢橋 美紀) gesprochen.
 
-48cm
+## Community
 
-#### Einsatz in
+### Aiko
 
-- Taiko (Skin)
+![Aiko](img/Aiko.png "Aiko")
 
-{Official skin for [Options's trigger](/wiki/Client/Options#Skin)}
+Aiko wurde von [JMC](https://osu.ppy.sh/users/774010) entworfen und war eine der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Sie ist ein energiegeladenes Mädchen mit einer Leidenschaft für osu!taiko, auch wenn sie nicht die Beste darin ist! Mit einem Paar modischer "Tabi"-Schuhe und einem Gefolge von pippidon-Accessoires lebt das Maskottchen von einst in diesem temperamentvollen Mädchen weiter. Sie ist mit nur 154 Zentimetern recht klein und wurde am 06.04.1999 geboren.
 
-#### Ersteller
+### Alisa
 
-- Yukiko Yokoo (横尾有希子) \[Aussehen\]
-- Narahashi Miki (楢橋 美紀) \[Stimme\]
+![Alisa](img/Alisa.png "Alisa")
 
-#### Geschlecht
+Alisa wurde von [\[ Glitch \]](https://osu.ppy.sh/users/3781400) entworfen und war eine der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Sie spielt osu!taiko schon seit ihrer frühen Kindheit. Wenn sie nicht gerade schläft oder isst, spielt sie osu!taiko oder hat Spaß an Retro-Videospielen!
 
-männlich
+### Chirou
 
-#### Beigetreten
+![Chirou](img/Chirou.png "Chirou")
 
-May 2008
+Chirou wurde von [pyun](https://osu.ppy.sh/users/981534) entworfen und war eine der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Sie ist sehr streng und anspruchsvoll, eine Perfektionistin und macht nicht gerne Fehler—vor allem nicht bei ihren Beats in osu!taiko. Aber hinter ihrer strengen Fassade kann sie, wenn man an ihren weichen Punkt kommt, auch flauschig und süß sein. Sie ist 14 Jahre alt und wurde am 25. Oktober mit der Blutgruppe AB geboren. Außerdem ist sie 1,80 m groß und wiegt 45 Kilogramm. Zu ihren Hobbys gehören das Trommeln, sich in ihrem Umhang warmzuhalten und Edelsteine oder Steine zu sammeln. Chirou hat einige Auftritte in verschiedenen Fanart-Beiträgen von Benutzern.
```

Comment:
Didn't know about this before, but [i looked it up](https://www.duden.de/sprachwissen/sprachratgeber/Infinitiv-mit-zu) and apparently you write words like `warmzuhalten` in one word. the more you know

Reply:
for `kennenzulernen` is [both possible](https://www.duden.de/rechtschreibung/kennenlernen)